*nl.NetlinkSocket does not implement nlReceiver (wrong type for Receive method)

Hi,
on my machine I get the following error when I import `barista.run/modules/netinfo`:

```
../../go/src/barista.run/base/watchers/netlink/nl.go:82:2: cannot use *nl.NetlinkSocket value as type nlReceiver in return argument:
      *nl.NetlinkSocket does not implement nlReceiver (wrong type for Receive method)
          have Receive() ([]syscall.NetlinkMessage, *unix.SockaddrNetlink, error)
          want Receive() ([]syscall.NetlinkMessage, error)
```

--- a/base/watchers/netlink/nl.go
+++ b/base/watchers/netlink/nl.go
@@ -75,7 +75,7 @@ var newNlRequest = func(proto, flags int) nlRequest {
 }
 
 type nlReceiver interface {
-	Receive() ([]syscall.NetlinkMessage, error)
+	Receive() ([]syscall.NetlinkMessage, *unix.SockaddrNetlink, error)
 }
 
 var nlSubscribe = func(protocol int, groups ...uint) (nlReceiver, error) {
@@ -136,7 +136,7 @@ func nlListen() {
 		return
 	}
 	for {
-		msgs, err := s.Receive()
+		msgs, _, err := s.Receive()
 		if err != nil {
 			l.Log("nl Receive failed: %s", err)
 			continue

--- a/base/watchers/netlink/nl_test.go
+++ b/base/watchers/netlink/nl_test.go
@@ -28,12 +28,12 @@ type testNlSubscriber struct {
 	errorChan <-chan error
 }
 
-func (t *testNlSubscriber) Receive() ([]syscall.NetlinkMessage, error) {
+func (t *testNlSubscriber) Receive() ([]syscall.NetlinkMessage, *unix.SockaddrNetlink, error) {
 	select {
 	case m := <-t.msgChan:
-		return []syscall.NetlinkMessage{m}, nil
+		return []syscall.NetlinkMessage{m}, nil, nil
 	case e := <-t.errorChan:
-		return nil, e
+		return nil, nil, e
 	}
 }
 

--- a/go.mod
+++ b/go.mod
@@ -16,8 +16,7 @@ require (
 	github.com/maximbaz/yubikey-touch-detector v0.0.0-20190725083449-25eabf6f1933
 	github.com/spf13/afero v1.2.2
 	github.com/stretchr/testify v1.4.0
-	github.com/vishvananda/netlink v1.0.0
-	github.com/vishvananda/netns v0.0.0-20190625233234-7109fa855b0f // indirect
+	github.com/vishvananda/netlink v1.1.0
 	github.com/zalando/go-keyring v0.0.0-20190913082157-62750a1ff80d
 	go.opencensus.io v0.22.1 // indirect
 	golang.org/x/crypto v0.0.0-20191029031824-8986dd9e96cf

